docs/usage.md

```diff
@@ -65,175 +32,231 @@ results         # Finished results (configurable, see below)
-```
+The path to a TSV file containing the peptide sequences. As a minimum, the peptide ID and sequence columns are required.
 
-It is not possible to run a mixture of single-end and paired-end files in one run.
+## Additional pipeline parameters
 
+## `--differential_gene_expression`
 
-## Reference Genomes
+The path to a differential gene expression file for additional annotation of results.
 
-The pipeline config files come bundled with paths to the illumina iGenomes reference index files. If running with docker or AWS, the configuration is set up to use the [AWS-iGenomes](https://ewels.github.io/AWS-iGenomes/) resource.
+## `--filter_self`
 
-### `--genome` (using iGenomes)
-There are 31 different species supported in the iGenomes references. To run the pipeline, you must specify which to use with the `--genome` flag.
+Specifies that peptides should be filtered against the specified human proteome references. By default, this is turned off.
 
-You can find the keys to specify the genomes in the [iGenomes config file](../conf/igenomes.config). Common genomes that are supported are:
+## `--gene_expression`
 
-* Human
-  * `--genome GRCh37`
-* Mouse
-  * `--genome GRCm38`
-* _Drosophila_
-  * `--genome BDGP6`
-* _S. cerevisiae_
-  * `--genome 'R64-1-1'`
+Specifies a path to a gene expression file for additional annotation of the obtained results.
 
-> There are numerous others - check the config file for more.
+## `--ligandomics_identification`
 
-Note that you can use the same configuration setup to save sets of reference files for your own use, even if they are not part of the iGenomes resource. See the [Nextflow documentation](https://www.nextflow.io/docs/latest/config.html) for instructions on where to save such a file.
+Specifies a path to a ligandomics identification results matrix, to allow for additional annotation of obtained predictions.
 
-The syntax for this reference configuration is as follows:
+## `--mhc_class`
 
-```nextflow
-params {
-  genomes {
-    'GRCh37' {
-      fasta   = '<path to the genome fasta file>' // Used if no star index given
-    }
-    // Any number of additional genomes, key is used with --genome
-  }
-}
-```
+Specifies whether the predictions should be done for MHC class I or class II. By default, this is set to 1 (class I).
 
-### `--fasta`
-If you prefer, you can specify the full path to your reference genome when you run the pipeline:
+## `--peptide_length`
 
-```bash
---fasta '[path to Fasta reference]'
-```
+Specifies the maximum peptide length. By default, for MHC Class I this is 8 to 11 amino acids. For MHC Class II this is regularly 15 to 16 amino acids.
+
+## `--protein_quantification`
+
+Add a path for a MaxQuant protein quantification table to annotate predictions with protein quantification results.
+
+### `--reference_genome`
+
+This defines against which reference genome the pipeline performs the analysis. The default choice is `GRCh37`. Available are `GRCh37` and `GRCh38`.
```

Comment:
Default is `GRCh37?
Why not `GRCh38`?

Reply:
Clarified this :-)

Reply:
From my experience `GRCh37` is still used more often. But we could also just set this to the most recent version.

Reply:
I have seen plenty people still using `GRCh37`, but I do thinik it's getting slightly too old.
So I do think we should encourage people to use `GRCh38` by default.